src/xcode/ENA/ENAUITests/CheckIns/ENAUITests_10_CheckIns.swift

```diff
@@ -80,12 +80,23 @@ class ENAUITests_10_CheckIns: XCTestCase {
 		
 		XCTAssertTrue(app.staticTexts[AccessibilityIdentifiers.Checkin.Information.descriptionTitle].exists)
 		XCTAssertTrue(app.staticTexts[AccessibilityIdentifiers.Checkin.Information.descriptionSubHeadline].exists)
-
-		XCTAssertTrue(app.cells[AccessibilityIdentifiers.Checkin.Information.dataPrivacyTitle].waitForExistence(timeout: .extraLong))
-
-		app.cells[AccessibilityIdentifiers.Checkin.Information.dataPrivacyTitle].tap()
 		
-		XCTAssertTrue(app.staticTexts["AppStrings.AppInformation.privacyTitle"].waitForExistence(timeout: .extraLong))
+		// find data privacy cell (last cell) and tap it
+		guard let lastCell = app.tables.firstMatch.cells.allElementsBoundByIndex.last,
+			  lastCell.identifier == AccessibilityIdentifiers.Checkin.Information.dataPrivacyTitle else {
+			XCTFail("Could not find last table view cell")
+			return
+		}
+		
+		let maxTries = 10
+		var actualTry = 0
+		while lastCell.isHittable == false && actualTry < maxTries {
+			app.swipeUp()
+			actualTry += 1
+		}
+		lastCell.tap()
```

Comment:
Do we really have to swipe through? If we do have the `AccessibilityIdentifier` we can omit the swiping:
```
let cell = app.cells[$lastCellsID]
XCTAssertTrue(waitForExistence(timeout: .short)) // not sure if needed
cell.tap()
```

Reply:
Normally I would agree, but before we used it and it did not work properly.

Reply:
I agree, wait for existence not always works, somehow, the whole eco-system around UI testing is unfortunately a mess.

Reply:
I think it's a bug on our side but if this works atm, let's keep it.